charts/brigade-project/values.yaml

```diff
@@ -92,3 +92,12 @@ kubernetes:
   # OPTIONAL: Override the storage class used for build storage. This must
   # point to a readWriteMany volume provisioner.
   #buildStorageClass: default
+
+###
+# ADAVANCED
+###
+
+# OPTIONAL: specify an alternative command that the worker pod should run on startup.
+# For example, if you want the typescript compiler to re-run, replace this with
+# 'yarn build-start'
```

Comment:
Just my two cents but also mentioning the two points below would be even more helpful:

- "The default is `yarn -s start` which suppresses log messages but the script output"
- "For extra log verbosity, just uncomment the line below"